### PR TITLE
Alpine Edits

### PR DESCRIPTION
**Pull request by pbzona**
Most changes here involve tweaking to conform more closely to our style and formatting guides. However, I also added some copy in a few places. Specifically:
-   High-level explanations in the section where we modify system files (fstab, inittab, et al.)
-   A short security section at the bottom, following the package notes
-   Various cautionary notes throughout. I think this is straightforward enough for any intermediate user to follow along with no problems, but as a custom distro it is technically unsupported.

Overall really nice job! Aside from the initial problem with networking I was able to run through the guide with very few bumps. Please feel free to let me know if you have questions on any of my changes. 

NOTE: The merge conflict commit came following your fixes re: the networking issue I had. I'd made some changes by that point and git didn't want to pull your commits in right away, but it's good to go now.

### CODE REVIEW

File: docs/tools-reference/custom-kernels-distros/install-alpine-linux-on-your-linode.md [hunk 1]
```diff
@@ -2,237 +2,305 @@
...
+## Install Alpine Linux
 
-### Boot into Recovery
+### Boot into Rescue Mode
 
-1.  From the Linode Manager, reboot your linode into recovery mode, with your boot disk image as /dev/sda, your root disk image as /dev/sdb, and your swap as /dev/sdc.
+1.  From the Linode Manager, boot your Linode into [rescue mode](https://www.linode.com/docs/troubleshooting/rescue-and-rebuild), with your boot disk image as `/dev/sda`, your root disk image as `/dev/sdb`, and your swap as /`dev/sdc`.
 
-2.  Using Lish, connect to the linode.  (If you are not familiar with Lish, there is a simple web interface for it located under Remote Access)
+2.  Once the Linode has booted, connect to it via Lish. If you are not familiar with Lish, there is a simple web interface for it located under the **Remote Access** in the Linode Manager.
```
Comment (andyleap) [inline docs/tools-reference/custom-kernels-distros/install-alpine-linux-on-your-linode.md]: "under the Remote Access tab"?  "under Remote Access"?

File: docs/tools-reference/custom-kernels-distros/install-alpine-linux-on-your-linode.md [hunk 2]
```diff
@@ -2,237 +2,305 @@
...
+
+4.  Download and extract the `apk-tools-static` package to your Linode. You should still be working in the `/alpine` directory when performing this step. Replace `address` in the following command with the address you copied in the previous step:
+        
+        curl -s address | tar xz
+
+5.  Perform the initial distro installation. This will use the latest stable build of Alpine:
 
-5.  Perform initial distro install.  This will preform the install using the latest stable build of Alpine.
-        ./sbin/apk.static --repository https://nl.alpinelinux.org/alpine/latest-stable/main/ --update-cache --allow-untrusted --root /alpine --initdb add alpine-base alpine-mirrors
+        ./sbin/apk.static --repository http://nl.alpinelinux.org/alpine/latest-stable/main/ --update-cache --allow-untrusted --root /alpine --initdb add alpine-base alpine-mirrors
```
Comment (andyleap) [inline docs/tools-reference/custom-kernels-distros/install-alpine-linux-on-your-linode.md]: Looking at this again, one of the things is that the repository url is supposed to be the url they picked earlier, should this get changed to reflect that, like the download apk-tools-static stuff?  Also, it got reverted to http at some point, would be better to be https.

**Comment by pbzona** [comment]
@andyleap Sorry for the wait, I was unexpectedly out of the office for a few days. Thanks for your changes, I've implemented them in a new commit. Please review and merge, and we can get things moving forward.

**Comment by andyleap** [comment]
Looks good to me, merging

**Pull request merged**